src/main/java/org/spongepowered/mod/SpongeGame.java

```diff
@@ -109,11 +111,14 @@ public ServiceManager getServiceManager() {
     }
 
     @Override
-    public Scheduler getScheduler() {
+    public SynchronousScheduler getSyncScheduler() {
         return SyncScheduler.getInstance();
     }
 
     @Override
+    public AsynchronousScheduler getAsyncScheduler() { return AsyncScheduler.getInstance(); }
```

Comment:
I'm not sure who started this but keeping everything on one line looks extremly weird, add the line breaks after the braces